### DISCUSSION TITLE
CHATGPT API（降价 90%）对 LLM 领域的影响 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#16] [General]
# CHATGPT API（降价 90%）对 LLM 领域的影响 | 高策

江湖小虾米

http://gaocegege.com/Blog/chatgpt-api

**Comment by t1nkerr** [comment]
巨大的算力成本会不会导致对应的安卓无法出现？

**Reply by gaocegege** [reply]
我觉得是早晚的事情，从市场竞争的角度来看

**Reply by t1nkerr** [reply]
嗯，我的意思是出现的可能不会是安卓，更可能是另一个苹果。OpenAI从开源nonprofit转变成苹果也是承受不了算力成本吧。
